hs_core/hydroshare/utils.py

```diff
@@ -464,6 +474,74 @@ def copy_resource_files_and_AVUs(src_res_id, dest_res_id, set_to_private=False):
             else:
                 istorage.setAVU(dest_coll, avu_name, value)
 
+    # link copied resource files to Django resource model
+    res_id_len = len(src_res.short_id)
+    files = ResourceFile.objects.filter(object_id=src_res.id)
```

Comment:
Can get the list of files as src_res.files.all() without the need for the filter.